Documentation/UserTsconfig/Options.rst

```diff
@@ -49,8 +50,8 @@ alertPopups
 -----------
 
 ..  confval:: alertPopups
-
-    :Data type: bitmask
+    :name: useroptions-alertPopups
+    :type: bitmask
```

Comment:
Why changing to "type"? We use "Data type" in TypoScript and TSconfig throughout the manual?

Reply:
type is a native property of the confval and always displayed on top. We can change its label globally if desired.

`:Data type:` is just an arbitary label without meaning behind it. Whereever I have seen it I have changed it or introduced the new one right away. And we can make a mass-change to catch those not caught in the end. Same goes with switching from definition list (with empty line after confval) to real parameters.

Reply:
Okay, so you take care about changing all the "Data type" to "type" in the next days? As this wasn't announced in the PR description, I commented about it.

Reply:
I have been unifying the confvals all over these  changes. This is one of the many inconsitencies I found. All my PRs had these kind of changes in them. It is an iterative process of getting them all on the same level. But as you can see I am working on it.

The way that Parameters where until now displayed in the TSconfig was inconsistent all over

Reply:
I just checked, there are currently 102 usages of `:type:` and only 44 usages of `:Data type:` in this reference. All usages of the latter are on the page this PR adresses

Reply:
Okay, thanks, TypoScript has far more "Data type" usages.